### ISSUE TITLE
Poor Speech output with JAWS Screen Reader

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#20181]
### Issue type

UX/Interaction bug (incorrect behaviour)

### Bug description

When editing a score with NVDA, the score is read accurately, for example: Note C 5 crotchet, , bar 1 beat 1 staccato above staccato above

(Note the double comma and the repetition of the articulation)

However, with JAWS, the output is:
Segment, bar 1 beat 1, chord automatic,, Note C 5 crotchet, bar 1 beat 1

Expected behaviour: The output should be the same in all screen readers and in the order of most important info first, e.g. note name, duration, bar, etc (as it is in NVDA)

Actual output: Note the verbose mention of segment, chord automatic, and so on, and the omission of the articulation.

The omissions in JAWS include some very important items such as not reading accidentals.

### Steps to reproduce

1.  Start Jaws screen reader.
2. Open any score with a variety of notes, accidentals, articulations, chord symbols, lyrics, expression and other elements.
3. Use right arrow to move along the score hearing what the screen reader says for each note.
4. Quit Jaws and start NVDA.
5. Repeat the same test to compare what the output should sound like.

### MuseScore Version

4.1.1.232071203

### Regression

I don't know

### Operating system

Windows 10 & 11

**Comment by NoahTCarver** [comment]
I can reproduce this issue in MuseScore version 4.1.1.232071203, revision e4d1ddf. I tested this against JAWS version 2024.2310.70, but this has been present in all versions of MuseScore 4 and JAWS 2022/2023/2024 which I have used. Unfortunately, this bug is so severe that I have to use NVDA with MuseScore.
